website/README.md

```diff
@@ -46,7 +46,7 @@ The website can be run locally through node.js or [Docker](https://www.docker.co
 
 ### With Docker
 
-Running the site locally is simple. Provided you have Docker installed, clone this repo, run `make`, and then visit `http://localhost:3000`.
+Running the site locally is simple. Provided you have the [latest version](https://docs.docker.com/engine/release-notes/) of Docker installed, clone this repo, run `make`, and then visit `http://localhost:3000`.
```

Comment:
Isn't this section just about building the website?  That's not impacted by the Go change is it?

Reply:
I put this in here to keep our references to Docker installation consistent. If you think it doesn't belong, I can drop it, but I just saw it as a potential entrypoint to Vault + Docker.

The README recommends that you clone the repo and run `make` (no mention of "run make in the website dir"). If someone decided to then use that build with the older version of Docker and things went sideways, they might have some indication as to why.

Reply:
Fair enough.